api-2.2/src/main/java/org/openmrs/module/fhir/helper/ConditionHelperImpl2_2.java

```diff
@@ -65,6 +68,48 @@ public Condition createCondition(Condition condition) {
 		return generateFHIRCondition(openMrsCondition);
 	}
 
+	/**
+	 * @see org.openmrs.module.fhir.api.helper.ConditionHelper#getConditionsByPatientUuid(java.lang.String)
+	 */
+	@Override
+	public List<Condition> getConditionsByPatientUuid(String patientUuid) {
+		List<Condition> fhirConditions = new ArrayList<>();
+		PatientService patientService = Context.getPatientService();
+		ConditionService conditionService = Context.getService(org.openmrs.api.ConditionService.class);
+
+		if (StringUtils.isNotBlank(patientUuid)) {
+			conditionService.getActiveConditions(
+					patientService.getPatientByUuid(patientUuid))
+					.forEach(condition -> fhirConditions.add(generateFHIRCondition(condition)));
+		}
+
+		return fhirConditions;
+	}
+
+	/**
+	 * @see org.openmrs.module.fhir.api.helper.ConditionHelper#getConditionsByPatientIdentifier(java.lang.String)
+	 */
+	@Override
+	public List<Condition> getConditionsByPatientIdentifier(String identifier) {
```

Comment:
Do you really have a use case for this?

Reply:
Not really.
But other resources have implemented. Thought could be useful
Anyway, got rid of it.

Reply:
We have had some code in the openmrs platform, that no one ever got to use. Basing on this background, the less code we have to maintain, the better. :)

Instead of having nice to haves, we rather code them in when the time comes to actually need them. And since modules have shorter release cycles, then it is easy to add any needed functionality and we release right away. But if the squad has a use case for it, then am fine leaving it in. 😊

Reply:
👍🏽